wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -6,6 +6,7 @@ import { Link } from "react-router-dom";
 import './Lobby.css';
 
 // change to http://localhost:5000 for local development
+// change to https://wewatch-server.herokuapp.com/ for deployment
```

Comment:
Good thinking adding this comment. I would change it to production deployment over just deployment just to make things more clear.